qln/break.go

```diff
@@ -40,6 +40,21 @@ func (nd *LitNode) BreakChannel(q *Qchan) error {
 
 	// set delta to 0... needed for break
 	q.State.Delta = 0
+
+	for i, h := range q.State.HTLCs {
+		if !h.Cleared && h.Clearing {
+			q.State.HTLCs[i].Clearing = false
+		}
+	}
+
+	if q.State.InProgHTLC != nil {
```

Comment:
You don't need to do a branch here, if it's already nil then you won't do anything.